packages/flutter/lib/src/material/button_bar.dart

```diff
@@ -126,6 +127,14 @@ class ButtonBar extends StatelessWidget {
   /// If that is null, it will default [ButtonBarLayoutBehavior.padded].
   final ButtonBarLayoutBehavior layoutBehavior;
 
+  /// Defines the vertical direction of a [ButtonBar]'s children if it
```

Comment:
Just saying that we're defining children's vertical direction is a little vague. Could explain this procedurally: if the dialog's [actions] don't fit on a single row, then they're arranged in a column where the first action is at the bottom of the column if ...